web/html/src/core/spa/view-helper.ts

```diff
@@ -13,6 +13,7 @@ const BOOTSTRAP_READY_PAGES: string[] = [
   "/rhn/multiorg/OrgConfigDetails.do",
   "/rhn/manager/notification-messages",
   "rhn/channels/software/Search.do",
+  "/rhn/manager/schedule/maintenance/schedules#/create",
```

Comment:
Is this hash fragment required here? I haven't tested it, but I would expect it to work without it since it's not a part of the path itself?